### DISCUSSION TITLE
Coloured taskbar

### DISCUSSION THREAD

**Discussion by NikolaasD** [SafeExamBrowser/seb-win-refactoring#1059] [Ideas]
Hello

When taking exams, our students are in a classroom with a teacher keeping an eye on them. I often get the feedback that at the start of an exam period, it is difficult to see who has correctly opened the Safe Exam Browser (because they are distributing exam copies, ...).

Would it be possible to give the option to have a coloured edge or taskbar when the SEB is opened?

**Comment by dbuechel** [comment]
I'm afraid I don't quite understand the use case: How would a custom taskbar color indicate that SEB has been opened with the correct configuration or alike?

For monitoring and similar use cases, we strongly recommend the usage of [SEB Server](https://github.com/SafeExamBrowser/seb-server), which (among many other things) contains an automatic client signature verification using the App Signature Key and now also provides a screen proctoring feature.
